homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -69,11 +71,11 @@ def __init__(self, remote, name, device_info):
         self._device_info = device_info
 
     @property
-    def unique_id(self) -> str:
+    def unique_id(self):
         """Return the unique ID of the device."""
-        if self._device_info is not None:
-            return self._device_info[ATTR_UDN]
-        return None
+        if self._device_info is None:
+            return None
+        return self._device_info[ATTR_UDN]
```

Comment:
There is still a UUID in [the response by Tom](https://github.com/home-assistant/core/issues/42205#issuecomment-715439181), isn't that the same as UDN ?

Reply:
actually the problem is that his TV apparently has no model number explicit in the descriptor file haha